ludwig/combiners/combiners.py

```diff
@@ -60,6 +59,35 @@ def wrap(cls):
     return wrap
 
 
+def get_combiner_jsonschema():
```

Comment:
Let's move this into a separate file (can be done in a follow-up PR).

Reply:
In description follow-up list